### DISCUSSION TITLE
v1.23 feedback thread

### DISCUSSION THREAD

**Discussion by Lailloken** [Lailloken/Exile-UI#50] [Ideas & Feedback]
Highlighted changes:
- settings menu
- LLK-UI panel
- notepad feature
- archnemesis scan-hotkey
- countdown/alarm timer feature
- omni-key
- context-menu for items

**Comment by antoniocmf** [comment]
I really liked the timer feature, and while using it I thought about another similar feature that would really help while mapping in my opinion, a map timer. Perhaps something like: https://github.com/mapwatch/mapwatch, which sadly doesn't work for me anymore.

**Reply by Lailloken** [reply]
Crazy stuff, never heard of that. I'll add that to the list of potential feature concepts, but I personally don't really care about playing fast, so my 'idea-horizon' is kind of limited when it comes to working on this.

**Reply by Lailloken** [reply]
This is an old request, but in case you're still interested and using my tool: v1.28.7 now has a mapping-tracker feature.

**Comment by bfugs22** [comment]
Hey so I've not used for a while, but decided to update, but since I've updated, I've not been able to save scans?  Every time I hit the scan button, it asks me to scan every single tile again.

**Reply by Lailloken** [reply]
Weird. So it's asking you to specify every icon over and over? Please do this to troubleshoot: Check which mod is in the first inventory slot (top-left corner), right-click the (?) label in the ready window to open the image folder. Check the image files of that mod -- are there multiple? If so, how many? -- scan and specify it again when asked, press cancel if it wants you to specify the next tile. Check the folder again to see what's changed for the first mod.
I don't remember off the top of my head if a scanner change has ever required to redo training, but it may well be possible depending on how long ago you used the script the last time. If it keeps asking you to re-specify the first tile, try deleting every image file of that mod and scan again to see how it behaves then.

**Reply by bfugs22** [reply]
So it adds a new image every time I hit scan and start over.  For example.  I have an echoist in top left.  It will add echoist, echoist1, echoist 2.

**Reply by Lailloken** [reply]
Do the images look OK? Is the icon more or less centered and clearly visible? Have you changed anything in your setup since using the script the last time, e.g. graphical filters or sharpening?

**Comment by bfugs22** [comment]
Yeah everything looks fine. I even tried deleting everything and redownload
and it still behaves as I described.

**Reply by Lailloken** [reply]
You could try out v1.21.5 in a separate folder. The scanner was changed significantly in the release after that (1.22), so that could be a way to check if that's the cause. You could also try updating AutoHotkey, that has solved similar issues for people in the past.

**Comment by OmikronGamma** [comment]
Hi, is there a way to move on-screen-LLKUI-button (the one in the top left) to some other place?

**Reply by Lailloken** [reply]
You can drag it like the other on-screen buttons.

**Reply by OmikronGamma** [reply]
Oh yeah, works. More often than not it opens up main menu, i guess thats what confused me.
